io/jvm-native/src/main/scala/fs2/io/process/ProcessesPlatform.scala

```diff
@@ -29,9 +29,16 @@ import cats.syntax.all._
 import fs2.io.CollectionCompat._
 
 import java.lang
+import java.util.concurrent.Executors
+import java.util.concurrent.ExecutorService
+import scala.concurrent.ExecutionContext
 
 private[process] trait ProcessesCompanionPlatform {
   def forAsync[F[_]](implicit F: Async[F]): Processes[F] = new UnsealedProcesses[F] {
+
+    private def javaVersion: Int =
```

Comment:
Is there a centralized place for such things?

Reply:
Hmm, maybe you could put this stuff here?
https://github.com/typelevel/fs2/blob/3b56691a0830580ea113ef391c2fc169a6f17b97/core/jvm/src/main/scala/fs2/internal/ThreadFactories.scala#L31

Reply:
Is this a place to move `javaVersion`? I wasn't explicit but that I was asking specifically for `javaVersion`.

Reply:
It's a place where we can move the `lazy val` for the virtual thread executor, not just `javaVersion`.